src/Tests/dotnet-sln.Tests/GivenDotnetSlnAdd.cs

```diff
@@ -1225,5 +1225,68 @@ public void WhenSolutionFolderIsPassedWithDirectorySeparatorFolderStructureIsCor
 
             return slnContents;
         }
+
+        [Fact]
+        public void WhenSolutionIsPassedAsProjectItPrintsSuggestionAndUsage()
```

Comment:
This is somewhat of a nit.. but all 15 lines of these three (really four) tests are identical minus the arguments passed in to them. I think we should instead make a helper method that just takes in the execute arguments and expected error message? Or you could turn this into a theory. I prefer the helper method because it lets you keep the better test names.